src/layouts/default.layout.scss

```diff
@@ -5,18 +5,23 @@
 @use '../styles/themes/light';
 @use '../styles/themes/dark';
 
+@use '../styles/colors';
+
 @include sourceSans.fontFace($weight: 400, $fontDir: '~/@fontsource/source-sans-pro/files');
 @include sourceSans.fontFace($weight: 600, $fontDir: '~/@fontsource/source-sans-pro/files');
 
 :root {
-  @include base.theme;
-  
-  // default theme
+  @include colors.colors;
+
+  @include base.theme;  
+}
+
+:root[theme="light"] {
```

Comment:
So either using `document.querySelector('html')` along with `html[theme="light"]`, \
_or_ on root node `document.documentElement` with `:root[theme="light"]`.

We should avoid mixing those, as the API does not always resolve to `html` and is a bit implicit here (non-empty) - Please stick to one pattern.

Reply:
I've changed the selector in `theme-switch.component.astro` to `document.documentElement`